numba/core/errors.py

```diff
@@ -493,6 +493,12 @@ class UnsupportedError(NumbaError):
     pass
 
 
+class RewriteUnsupportedError(UnsupportedError):
```

Comment:
Wondering if `UnsupportedRewriteError` sounds better?

Reply:
it does

Reply:
renamed